bot/seasons/easter/egg_hunt/cog.py

```diff
@@ -319,26 +325,33 @@ def current_timestamp() -> int:
                 await asyncio.sleep(remaining)
 
             log.debug(f"Hunt started.")
-            current_window = EggHuntSettings.start_time
-            next_window = 0
-            for window in EggHuntSettings.windows:
-                window = int(window)
-                if window < now:
-                    current_window = window
+
+            db = sqlite3.connect(DB_PATH)
+            c = db.cursor()
+
+            current_window = None
+            next_window = None
+            windows = EggHuntSettings.windows.copy()
+            windows.insert(0, EggHuntSettings.start_time)
+            for i, window in enumerate(windows):
+                c.execute(f"SELECT COUNT(*) FROM super_eggs WHERE window={window}")
+                alread_dropped = c.fetchone()[0]
+                if alread_dropped:
                     continue
-                if not next_window:
-                    next_window = window
                 else:
+                    current_window = window
+                    next_window = window[i+1]
```

Comment:
```py
04/21/19 12:40:31 - asyncio ERROR: Exception in callback EggHunt.task_cleanup(<Task finishe...bscriptable")>)
handle: <Handle EggHunt.task_cleanup(<Task finishe...bscriptable")>)>
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/home/sebastiaan/pydis/seasonalbot/bot/seasons/easter/egg_hunt/cog.py", line 303, in task_cleanup
    task.result()
  File "/home/sebastiaan/pydis/seasonalbot/bot/seasons/easter/egg_hunt/cog.py", line 343, in super_egg
    next_window = window[i+1]
TypeError: 'int' object is not subscriptable
```

Did you mean `windows[i+1]`?
```suggestion
                    next_window = windows[i+1]
```